apis/dialapi/dialapi.yaml

```diff
@@ -1467,6 +1716,28 @@ components:
         metadata:
           type: string
           description: Additional metadata of dialcode
+    dialapiDialcodeGetV2:
+      title: Dialcode Get
+      type: object
+      properties:
+        identifier:
+          type: string
+          description: Represents identifier of dialcode
+        channel:
+          type: number
+          description: represents channel under which dialdoe is generated
```

Comment:
Please rectify the typo in "dialdoe" in the description and titlecase "represents".